localstack/services/apigateway/provider.py

```diff
@@ -515,6 +515,100 @@ def _delete_children(resource_to_delete: str):
         parent_id = moto_resource.parent_id
         api_resources[parent_id].remove(resource_id)
 
+    def update_integration(
+        self,
+        context: RequestContext,
+        rest_api_id: String,
+        resource_id: String,
+        http_method: String,
+        patch_operations: ListOfPatchOperation = None,
+    ) -> Integration:
+        # XXX: THIS IS NOT A COMPLETE IMPLEMENTATION, just the minimum required to get tests going
+        # TODO: validate patch operations
+
+        moto_rest_api = get_moto_rest_api(context, rest_api_id)
+        moto_resource = moto_rest_api.resources.get(resource_id)
+        if not moto_resource:
+            raise NotFoundException("Invalid Resource identifier specified")
+
+        moto_method = moto_resource.resource_methods.get(http_method)
+        if not moto_method:
+            raise NotFoundException("Invalid Method identifier specified")
+
+        integration = moto_method.method_integration
+
+        for patch_operation in patch_operations:
+            op = patch_operation.get("op")
+            path = patch_operation.get("path")
+            if "/requestParameters" not in path:
+                raise BadRequestException(
+                    f"Invalid patch path  '{path}' specified for op '{op}'. Must be one of: [/requestParameters]"
+                )
+            if op != "add":
+                raise BadRequestException(
+                    f"Invalid patch path  '{path}' specified for op '{op}'. Please choose supported operations"
+                )
+
+            # for path "/requestParameters/integration.request.header.Accept"
+
+            if "/requestParameters/integration.request.header" in path:
+                value = patch_operation.get("value")
+                if not isinstance(value, str):
+                    raise BadRequestException(
+                        f"Invalid patch value  '{value}' specified for op '{op}'. Must be a string"
+                    )
```

Comment:
does it need to have a `value` if it's a remove call?

Reply:
good catch, it does not, my test add a value, that's why it does not break. will fix it!